src/Utility/merge.luau

```diff
@@ -24,6 +24,8 @@ local function merge(
 				into[key] = value
 			elseif overwrite == "none" then
 				External.logError("mergeConflict", nil, tostring(key))
```

Comment:
If this is what the intended behavior was supposed to be and it should overwrite, including the **elseif** statement as a whole would be unnecessary.
```suggestion
			if overwrite == "none" then
				External.logError("mergeConflict", nil, tostring(key))
			else
				into[key] = value
			end
```

Reply:
We should only error if the key already exists, and would overwrite a value.